perception/image_projection_based_fusion/config/roi_detected_object_fusion.param.yaml

```diff
@@ -0,0 +1,18 @@
+/**:
+  ros__parameters:
+    # UNKNOWN, CAR, TRUCK, BUS, TRAILER, MOTORBIKE, BICYCLE, PEDESTRIAN
+    passthrough_lower_bound_probability_thresholds: [0.35, 0.35, 0.35, 0.35, 0.35, 0.35, 0.35, 0.50]
+    min_iou_threshold: 0.5
```

Comment:
@yukke42 Need to add trust_distances in param.yaml ?

Reply:
sorry, I missed it. [4d68604](https://github.com/autowarefoundation/autoware.universe/pull/4546/commits/4d68604595875814bc4eda886f10b8c497a18766)